src/window/gui/Gui.cpp

```diff
@@ -719,20 +717,20 @@ void Gui::ImGuiRenderDrawData(ImDrawData* data) {
             GX2SetScissor(0, 0, mImGuiIo->DisplaySize.x, mImGuiIo->DisplaySize.y);
             ImGui_ImplWiiU_DrawKeyboardOverlay();
             break;
-#else
+#elif defined(ENABLE_OPENGL)
```

Comment:
i'm having a hard time knowing where these blocks end when reviewing because of all the elifs, it's not the worst here (`ImGuiBackendInit` is rough) but if we instead had
```cpp
#ifdef __WIIU__
// gx2
#endif

#ifdef ENABLE_OPENGL
// sdl opengl
#endif
```
we could use whitespace to improve readability quite a bit